weblate/formats/tests/test_formats.py

```diff
@@ -1133,3 +1135,20 @@ class TBXFormatTest(AutoFormatTest):
     FIND_MATCH = "adresní řádek"
     NEW_UNIT_MATCH = b"<term>Source string</term>"
     EXPECTED_FLAGS = ""
+
+
+class FluentFormatTest(AutoFormatTest):
+    FORMAT = FluentFormat
+    FILE = TEST_FLUENT
+    BASE = ""
+    MIME = ""
```

Comment:
It should have a MIME type, based on https://gitlab.gnome.org/GNOME/gtksourceview/-/merge_requests/87/diffs?commit_id=df7a2a1638f9661a91d4d27ae71b2ce506fd7bcb:
```suggestion
    MIME = "text/x-fluent"
```